libs/app-localize/utils/src/lib/locales/en-US.json

```
@@ -93,6 +93,18 @@
   "general.welcomeScreen.troubleshootingButton.text": "My device doesn’t show up after connecting",
   "harmony.menu.files": "Manage Files",
   "harmony.menu.overview": "Overview",
+  "harmony.overview.os.title": "Mudita OS",
+  "harmony.overview.os.update.checkButton": "Check for updates",
+  "harmony.overview.os.versionLabel": "Current version:",
+  "harmony.overview.os.versionText": "Mudita OS {version}",
+  "harmony.overview.status.battery": "Battery",
+  "harmony.overview.status.title": "Status",
```

This looks like a regression in the production version (needs confirmation). The "Connected" badge is missing compared to production, the copy has changed from "Status" to "Harmony Status", and the styles also look slightly different (layout and shadows).

```suggestion
  "harmony.overview.status.title": "Harmony Status",
```